docs: Update text in Sharing code tutorial (#401)
sveltejs/learn.svelte.dev: content/tutorial/02-advanced-svelte/10-module-context/01-sharing-code/README.md

Very occasionally, you'll need to run some code outside of an individual component instance. For example: returning to our custom audio player from a [previous exercise](media-elements), you can play all five tracks simultaneously. It would be better if playing one stopped all the others.

> We can do that by declaring a `<script context="module">` block. Code contained inside it will run once, when the module first evaluates, rather than when a component is instantiated. Place this at the top of `AudioPlayer.svelte`, noting that we have to add a new script block as we only want the `current` variable as the only one affected by the context:

Reviewer: ```suggestion
We can do that by declaring a `<script context="module">` block. Code contained inside it will run once, when the module first evaluates, rather than when a component is instantiated. Place this at the top of `AudioPlayer.svelte` (note that this is a _separate_ script tag):
```
